cmd/gossamer/unlock.go

```diff
@@ -91,7 +91,7 @@ func unlockKeys(ctx *cli.Context, datadir string, ks *keystore.Keystore) error {
 		}
 
 		ks.Insert(kp)
-		log.Info("[cmd] unlocked key", "key", kp.Public().Hex())
+		log.Info("[gossamer] unlocked key", "key", kp.Public().Hex())
```

Comment:
in general this should be `[package name]` so `[cmd]`

Reply:
the directory name is `gossamer`...?

Reply:
ah true, the package name is `main` though

Reply:
oh boy, hehe, what should I use then?

Reply:
`main` would be good for consistency, but in this context `gossamer` is ok, you can leave it for now and let's try to figure out logging format soon